recipes/wub/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
   - tqdm
   - h5py
   - statsmodels
-  - htslib <1.4
+  - htslib {{ CONDA_HTSLIB }}*
```

Comment:
Looks like `wub` is a well-behaving Python-only packge. So we should only use
```yaml
requirements:
  build:
    - python
    - setuptools
```
for the build requirements.